database/feature-data.js

```diff
@@ -0,0 +1,396 @@
+  },
+  {
+    idString: 'luggageDropoff',
+    featureTitle: 'Luggage dropoff allowed',
+    featureDescr: 'For guests\' convenience when they have early arrival or late departure',
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'hangers',
+    featureTitle: 'Hangers',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'hairDryer',
+    featureTitle: 'Hair dryer',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'shampoo',
+    featureTitle: 'Shampoo',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'bedLinens',
+    featureTitle: 'Bed linens',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'extraBedding',
+    featureTitle: 'Extra pillows and blankets',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'bedroomLock',
+    featureTitle: 'Lock on bedroom door',
+    featureDescr: 'Private room can be locked for safety and privacy',
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'bbqGrill',
+    featureTitle: 'BBQ grill',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'gardenYard',
+    featureTitle: 'Garden or backyard',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'patioBalcony',
+    featureTitle: 'Patio or balcony',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'fireExting',
+    featureTitle: 'Fire extinguisher',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'co2Alarm',
+    featureTitle: 'Carbon monoxide alarm',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'smokeAlarm',
+    featureTitle: 'Smoke alarm',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  },
+  {
+    idString: 'firstAidKit',
+    featureTitle: 'First aid kit',
+    featureDescr: null,
+    iconUrl: 'https://visualpharm.com/assets/757/Briefcase-595b40b85ba036ed117daf4c.svg'
+  }
+];
+
+const insertFeatureData = function() {
+  Feature.create(featureData)
+    .then(() => db.disconnect());
```

Comment:
Did you get a TypeError: db.disconnect is not a function when running this?
I used this approach to fix the error: https://stackoverflow.com/questions/48481040/mongoose-disconnect-is-not-a-function:
"mongoose.connect does not return a value. If you want to close the database connection, you need to call the close method on mongoose.connection which is written to when you open a connection with mongoose.connect. The simplest way to do this is to move your mongoose.connect to its own line:
mongoose.connect(mongoURI);
const db = mongoose.connection;
Then just replace your calls to db.disconnect() with calls to db.close()."

Minh suggested to add catch block to this for error handling. I added both  just in case.

Reply:
thank you! i did not get this specific error but there's definitely some weirdness going on with my Mongoose model so I have a refactor ticket in my sprint plan. I will check out that SO link before I tackle it.